### DISCUSSION TITLE
Bunch keeps trying to open Teams in Parallels Windows VM

### DISCUSSION THREAD

**Discussion by ukmac** [ttscoff/bunch-docs#310] [General]
I want to start Teams as part of a work bunch but cant get it to open the macOS version, instead it keeps firing up Parallels and trying to launch the Windows version ?

**Comment by ttscoff** [comment]
Have you tried using the bundle ID instead of the app name?

https://bunchapp.co/docs/troubleshooting/#bundleid

Thanks,
Brett

**Reply by ukmac** [reply]
Yes I have tried that already. When I add bundleid nothing starts.

**Reply by ukmac** [reply]
I have sorted it. I added the Parallels Apps folder to Spotlight Privacy to exclude Spotlight from seeing them, this seems to have fixed it :)

**Comment by ttscoff** [comment]
Great tip!

Thanks,
Brett
